src/PackingList/PackingList.js

```diff
@@ -1,23 +1,46 @@
+import React, { useState, useEffect } from 'react'
 import StaticCategory from '../StaticCategory/StaticCategory'
 import Error from '../Error/Error'
+import { BiPencil } from 'react-icons/bi'
 import './PackingList.scss'
+import { deleteItem } from '../actions/actions'
 import { connect } from 'react-redux'
+import Modal from 'react-modal'
+import { customStyles } from './modalStyles'
 
-const PackingList = ({ packingList }) => {
+const PackingList = ({ packingList, deleteItem }) => {
 
   const { tripDetails, categories } = packingList
+  const [modalIsOpen,setIsOpen] = useState(false);
+  const [categoryToDelete, setCategoryToDelete] = useState('')
+  const [itemToDelete, setItemToDelete] = useState('')
+  const [verifyDeletion, setVerifyDeletion] = useState(true)
+  
+  useEffect(() => {
+    Modal.setAppElement('body');
```

Comment:
What is AppElement? Should we be more specific with this name?

Reply:
The modal needs an element to bind to so a screen reader can read it - i'm pretty sure that is the name of the method that does that! @jaypeasee

Reply:
Thanks for asking that!